tensorflow/core/platform/default/build_config.bzl

```diff
@@ -194,12 +194,14 @@ def tf_kernel_tests_linkstatic():
 def tf_additional_lib_defines():
   return select({
       "//tensorflow:with_jemalloc": ["TENSORFLOW_USE_JEMALLOC"],
+      "//tensorflow:linux_ppc64le":["TENSORFLOW_USE_JEMALLOC"],
```

Comment:
Here, we know that this change would always enable jemalloc to build irrespective of whether it is configured. If we don't do this, then code in tensorflow/BUILD that enables jemalloc only for k8 cpu will never allow jemalloc to be built on non-x86 arch. Kindly suggest appropriate way to handle ppc64le case too.